src/enochecker/enochecker.py

```diff
@@ -497,7 +361,8 @@ def run(self, method: Optional[str] = None) -> CheckerResult:
                 "Error in connection to service occurred: {}\n".format(ex), exc_info=ex
             )
             return CheckerResult(
-                Result.OFFLINE, message="Error in connection to service occured"
+                CheckerTaskResult.CHECKER_TASK_RESULT_OFFLINE,
```

Comment:
Isn't `CHECKER_TASK_RESULT_` redundant here and, hence, should be removed from the enum for clarity?

Reply:
this comes from `enochecker_core` and should be discussed/changed there, although that does not need to happen before this PR is merged imo